tikv/pd
# cluster: fix health metrics

Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?

- Fix https://github.com/tikv/pd/issues/2964
- Fix the issue that healthy value is 0 and unhealthy value is 1 (should be 1 and 0)

### What is changed and how it works?

- add metrics reset (after removing a member, you will still need to restart PD or transfer leader to reset metrics)
- fix wrong value

### Check List

Tests


- Manual test (add detailed scripts or steps below)
 
1. start 3pd+1tikv cluster
2. check /metrics output
```
pd_cluster_health_status{name="pd1"} 1
pd_cluster_health_status{name="pd2"} 1
pd_cluster_health_status{name="pd3"} 1
```
3. delete member pd3
4. transfer leader
5. check /metrics output again, removed member is not displayed
```
pd_cluster_health_status{name="pd1"} 1
pd_cluster_health_status{name="pd2"} 1
```

### Release note
- Fix the issue that member health metrics not correct
